packages/x-components/src/x-modules/facets-next/facets-service.types.ts

```diff
@@ -0,0 +1,70 @@
+import { Facet, Filter } from '@empathyco/x-types-next';
+
+/**
+ * An object containing a list of facets, and the group they belong to.
+ *
+ * The facet group is used to perform operations over a list of facets. For example, some APIs
+ * have dynamic facets. This means that depending on the selected filters, some facets can be
+ * returned. For example if you select `category:television`, a new facet called *Screen size* might
+ * appear. And, because now filters are stored in a dictionary instead of an array, and overridden
+ * with each request, when the user performs a new query, we have to remove all the `Screen size`
+ * filters because we don't know if the API will return them or not in the new query.
+ *
+ * @public
+ */
+export interface FacetGroup {
```

Comment:
Consider moving this to the search-types package.

Reply:
It shouldn't, I remember discussing several ways to set facets groups, but none of them was creating `FacetGroup` objects manually from TypeScript code.